tests/test-admin-locking.sh

```diff
@@ -25,10 +25,10 @@ set -euo pipefail
 setup_os_repository "archive-z2" "syslinux"
 
 # If parallel is not installed, skip the test
-if ! parallel --help >/dev/null 2>&1; then
-    echo "1..0 # SKIP no /usr/bin/parallel"
+if ! which parallel >/dev/null 2>&1; then
```

Comment:
(The reason this would be a regression is that moreutils parallel doesn't have `--help`, so more or less by coincidence, the test is correctly skipped if you have moreutils parallel rather than GNU parallel)